bower.json

```diff
@@ -1,22 +1,39 @@
 {
   "name": "lazy-imports",
+  "version": "2.0.0",
   "description": "Declarative lazy imports",
+  "authors": [
+    "The Polymer Authors"
+  ],
+  "keywords": [
+    "web-components",
+    "polymer",
+    "lazy",
+    "import"
+  ],
   "main": "lazy-imports.html",
-  "version": "2.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Polymer/lazy-imports.git"
+  },
+  "license": "https://polymer.github.io/LICENSE.txt",
+  "homepage": "https://github.com/Polymer/lazy-imports",
+  "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0-rc.3"
+    "polymer": "Polymer/polymer#^1.9 - 2"
   },
   "devDependencies": {
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#2.0-preview",
-    "paper-button": "PolymerElements/paper-button#2.0-preview",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#1 - 2",
+    "paper-button": "PolymerElements/paper-button#1 - 2",
     "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
-    "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.5",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.5"
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.10",
```

Comment:
Oh, nice! I checked earlier and it wasn't tagged yet. Will address comments in a minute. Thanks for taking a look and the feedback!